app/src/screens/SignIn/index.tsx

```diff
@@ -38,15 +40,22 @@ const styles = StyleSheet.create({
 });
 
 const SignIn: React.FC = () => {
-  const { authorize } = useAuth0();
+  const { authorize, getCredentials } = useAuth0();
+  const dispatch = useDispatch();
 
   const onLogin = async () => {
     try {
-      await authorize({ scope: config.auth0Scope });
+      await authorize({
+        scope: config.auth0Scope,
+        audience: config.audience,
+      });
+      const { accessToken } = await getCredentials();
```

Comment:
I think this sort of cascading action logic should belong in the store/actions file. Let's think of an async action that performs this sort of authentication -> token fetching logic in one action we can dispatch from this component.

Reply:
Actually, I can see the merit of two separate actions here. Let's move the authorize call into an action, though.

Reply:
Does Auth0 have utility functions for this sort of functionality outside of React hooks? Could we fabricate our own actions instead?

Reply:
That's a good point. My opinion is that this kind of logic seems very "atomic" in the sense we don't really need to use the steps individually anywhere in the app. The hook they provide seems more for apps that don't use centralized logic, but I can see your point.